### PR TITLE
Setup repo

### PR DESCRIPTION
**Pull request by pradel**
It is just to show you how simple is jest for testing and coverage and you don't need a lot of dependencies etc..
But it is just a test and I can delete this branch and setup with chai.
Can I have your thought on this? @TimMikeladze @davidyaha 

Still missing flow

### CODE REVIEW

**Comment by davidyaha** [comment]
I don't mind using jest.
Could you PR jest for the other repos as well?

**Comment by pradel** [comment]
Waiting for a review of @TimMikeladze and if he is okay I will do that

**Comment by TimMikeladze** [comment]
I'm cool with it.

**Comment by pradel** [comment]
I setup circleci to use yarn https://circleci.com/docs/install-and-use-yarn/
Should I make pr to other repo to do the same?

Are you okay to merge this pr into master?

File: circle.yml [hunk 1]
```diff
@@ -1,6 +1,27 @@
 machine:
   node:
     version: 6.2.1
+  environment:
```
Comment (TimMikeladze) [inline circle.yml]: Is there any reason to do a manual install of yarn for circle? If yarn is included as a dev dependency you should be able to run it from circle I believe.

Comment (pradel) [inline circle.yml]: Is to avoid the use for npm. It still need an npm install to install yarn as a dev dependency. So we loose the interest of yarn.

Comment (pradel) [inline circle.yml]: And we don't have yarn in dev dependencies as npm I think.

Comment (TimMikeladze) [inline circle.yml]: Got it.

**Pull request merged**